packages/block-library/src/categories/index.js

```diff
@@ -19,35 +19,11 @@ export const settings = {
 
 	category: 'widgets',
 
-	attributes: {
```

Comment:
To keep us on track here I'd really recommend preserving the definitions in JS — it keeps the change surface minimised as we careen toward code freezes.

Reply:
@chrisvanpatten If the definitions remain in JS but are removed from PHP, I can't stop invalid classes from being applied. Should I define the attributes in both JS and PHP?

Reply:
The JS should ensure that only valid alignments are set to the `align` attribute. It doesn't need to be reinforced in PHP (or to be more specific, we aren't currently enforcing it on PHP in this block and if an 'invalid' alignment somehow made it through to PHP it isn't a site-breaking issue).